kolibri/plugins/device/assets/src/views/DeviceIndex.vue

```diff
@@ -8,6 +8,14 @@
       <DeviceTopNav />
     </template>
 
+    <transition name="delay-entry">
```

Comment:
I think the designs call for this to be shown when clicking on the Device nav item, prior to navigation to the Device page, so we may need to include this in the DeviceSideNav also, and then persist the authentication state across the page reload with a cookie.

Reply:
Thanks @rtibbles! Yes, you are right about this. I did notice this during visual testing. Thanks for the hint!